app/views/revisions/show.html.erb

```diff
@@ -1,23 +1,35 @@
 <% content_for :title, "#{@record.class.model_name} ##{@record.id} - comparing revisions" %>
 
-<% if @node %>
-  <%= render "nodes/sidebar" %>
+<% content_for :breadcrumbs do %>
+  <%= render partial: 'breadcrumbs' %>
+<% end %>
+
+<% content_for :sidebar do %>
+  <% if @issue %>
+    <%= render 'issues/sidebar' %>
+  <% elsif @list %>
+    <%= render 'cards/sidebar' %>
+  <% elsif @node %>
+    <%= render 'nodes/sidebar' %>
+  <% end %>
 <% end %>
 
 <div class="row m-0">
   <div class="col-12 col-xl-8 col-xxl-9 p-0">
     <div class="content-container pb-3 mr-xl-3">
-      <div class="diff">
-        <h4 class="header-underline">
-          <%= @record.class.model_name %>
-          #<%= @record.id %> <span class="text-capitalize"> - view changes at <%= @diffed_revision.updated_at %></span>
-        </h4>
-        <pre class='js-diff-body'><%= @diffed_revision.diff.format_as(:color) %></pre>
-      </div>
+      <% if @diffed_revision %>
```

Comment:
With this change, we've lost all of our front-end caching. We need to bring this back here, or we'll be taking a perf hit.